tests/phpunit/AutoReview/Makefile/MakefileTest.php

```diff
@@ -204,16 +204,16 @@ public function test_phony_targets_are_correctly_declared(): void
 
     public function test_no_target_is_being_declared_twice(): void
```

Comment:
@theofidry 

How do you feel about renaming the test methods as well?

Reply:
Happy to in general, worth second look with fresh eyes, but I would argue _this specific one_ is fine IMO, unless you have a more suited one? (it feels replacing target to rule in this method incorrect since what matters is not the fact that a rule may be declared twice, but that the targets are – the rules pre-requisites and recipes could be different)